### PR TITLE
Avx functor globals

### LINKED ISSUE #135: intrinsic functor
**Is your feature request related to a problem? Please describe.**
performance not yet optimal (at most 60-70% of ls1 mardyn)

**Describe the solution you'd like**
use intrinsics in kernel

### PR DESCRIPTION
**Pull request by LG-TUM**
# Description

- [x] Implemented calculation of globals (virial and upot) for the AVX based Lennard-Jones-Functor.
- [x] Changed LJFunctorAVXTest st. it checks the correctness of the calculation of globals.

## Resolved Issues # (issue)

Missing calculation of globals in the LJFunctorAVX
closes #135 

# How Has This Been Tested?

ctest -R 'AVX' (Compares the results of the calculation of globals from the 'normal' LJFunctor and the LJFunctorAVX for the same scenario)

### CODE REVIEW

File: src/autopas/pairwiseFunctors/LJFunctorAVX.h [hunk 1]
```diff
@@ -68,11 +69,13 @@ class LJFunctorAVX : public Functor<Particle, ParticleCell, typename Particle::S
         _highCorner(highCorner),
         _postProcessed{false} {
     if (calculateGlobals or duplicatedCalculation) {
-      utils::ExceptionHandler::exception("Calculation of global values and duplicated calculations not supported.");
+      /*
+       *utils::ExceptionHandler::exception("Calculation of global values and duplicated calculations not supported.");
```
Comment (FG-TUM) [inline src/autopas/pairwiseFunctors/LJFunctorAVX.h]: If the exception is no longer needed remove also the `if`

File: src/autopas/pairwiseFunctors/LJFunctorAVX.h [hunk 2]
```diff
@@ -102,7 +113,11 @@ class LJFunctorAVX : public Functor<Particle, ParticleCell, typename Particle::S
     double *const __restrict__ fyptr = soa.template begin<Particle::AttributeNames::forceY>();
     double *const __restrict__ fzptr = soa.template begin<Particle::AttributeNames::forceZ>();
 
-    // @TODO: Globlas
+    __m256d virialSumX = _mm256_setzero_pd();
+    __m256d virialSumY = _mm256_setzero_pd();
+    __m256d virialSumZ = _mm256_setzero_pd();
+    __m256d upotSum = _mm256_setzero_pd();
+
     //    bool isHaloCell1 = false;
```
Comment (FG-TUM) [inline src/autopas/pairwiseFunctors/LJFunctorAVX.h]: Halo check is missing here

Comment (LG-TUM) [reply src/autopas/pairwiseFunctors/LJFunctorAVX.h]: Added in new commit.

File: src/autopas/pairwiseFunctors/LJFunctorAVX.h (hunk 1 above)
Comment (LG-TUM) [reply src/autopas/pairwiseFunctors/LJFunctorAVX.h]: Removed exception and `if`

Added exception from LJFunctor.h to check if lowCorner and highCorner are properly specified.

**Pull request merged**